docs/api/internal.json

```diff
@@ -82,19 +82,38 @@
                             "example": false
                           },
                           "job_type":  {
-                            "type": "string",
+                            "type": "array",
                             "enum": [
                               "freelance",
                               "full-time",
                               "internship",
                               "part-time",
                               "temporary",
                               "other"
-                            ]
+                            ],
+                            "items" : {
+                              "type": "string",
+                              "example": "Full Time"
+                            }
                           },
                           "salary":  {
-                            "type": "string",
-                            "example": "$50,000 - $100,000"
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "currency": {
+                                  "type": "string",
+                                  "example": "USD"
+                                },
+                                "value": {
+                                  "type": "string",
+                                  "example": "$50,000 - $100,000"
+                                },
+                                "unit": {
+                                  "type": "string",
+                                  "example": "YEAR"
```

Comment:
This could also be an `enum`, right?

Reply:
Fixed in c7dc9536e2935b9c52f42781fc9f23f99d60e14f